components/drawer-layout/DrawerMenuContent.tsx

```
@@ -13,33 +13,42 @@ import BrandActionItem from "./BrandActionItem";
 import { Searchbar } from "react-native-paper";
+import {
+  faHandshake,
+  faComment,
+  faStar,
+} from "@fortawesome/free-regular-svg-icons";
+import {
+  faFileSignature,
+  faPlusCircle,
+} from "@fortawesome/free-solid-svg-icons";
 
 interface DrawerMenuContentProps { }
```

Biome

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)